pkg/kubelet/kubelet.go

```diff
@@ -60,6 +60,7 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/bandwidth"
 	utilErrors "k8s.io/kubernetes/pkg/util/errors"
+	kubeioutil "k8s.io/kubernetes/pkg/util/io"
```

Comment:
ioutil to match the stdlib?  minor point really.

On Mon, Sep 14, 2015 at 9:17 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/13791#discussion_r39465928>
> :
>
> > @@ -60,6 +60,7 @@ import (
> >  	"k8s.io/kubernetes/pkg/util"
> >  	"k8s.io/kubernetes/pkg/util/bandwidth"
> >  	utilErrors "k8s.io/kubernetes/pkg/util/errors"
> > +	kubeioutil "k8s.io/kubernetes/pkg/util/io"
>
> utilio ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13791/files#r39465928>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift